packages/cli/src/lib/project/project.spec.ts

```diff
@@ -0,0 +1,113 @@
+jest.mock('@coveord/platform-client');
+jest.mock('fs');
+jest.mock('archiver');
+
+import {mocked} from 'ts-jest/utils';
+import {existsSync, createWriteStream, WriteStream, unlinkSync} from 'fs';
+import {Project} from './project';
+import {join} from 'path';
+import * as archiver from 'archiver';
+import {Writable} from 'stream';
+
+const mockedExistSync = mocked(existsSync);
+const mockedUnlinkSync = mocked(unlinkSync);
+const mockedCreateWriteStream = mocked(createWriteStream);
+const mockedArchiver = mocked(archiver);
+const mockedPipe = jest.fn();
+const mockedPassDirectory = jest.fn();
+const mockedFinalize = jest.fn();
+
+mockedArchiver.mockImplementation(
+  () =>
+    ({
+      on: () => {},
+      pipe: mockedPipe,
+      directory: mockedPassDirectory,
+      finalize: mockedFinalize,
+    } as unknown as archiver.Archiver)
```

Comment:
If you do my suggestion above
```suggestion
    } as unknown as Archiver)
```